crates/blockifier/src/debug.rs

```diff
@@ -0,0 +1,79 @@
+#![allow(clippy::as_conversions)]
+use std::collections::HashMap;
+use std::str::FromStr;
+
+use cairo_vm::vm::runners::cairo_runner::ExecutionResources;
+use starknet_api::transaction::TransactionHash;
+use starknet_core::types::MaybePendingTransactionReceipt::Receipt;
+use starknet_core::types::TransactionReceipt;
+use starknet_crypto::FieldElement;
+use starknet_providers::jsonrpc::HttpTransport;
+use starknet_providers::{JsonRpcClient, Provider, Url};
+use tokio::runtime::Runtime;
+
+pub fn get_execution_resources(tx_hash: TransactionHash) -> ExecutionResources {
+    println!("Getting execution resources for {}", tx_hash);
+
+    let rpc_url = std::env::var("RPC_URL").unwrap_or("http://localhost:6060".to_string());
+    let rpc_client = JsonRpcClient::new(HttpTransport::new(Url::from_str(&rpc_url).unwrap()));
```

Comment:
Does Juno need to run anything extra for performing this, or can it find quickly on its DB?